graph.go

```diff
@@ -267,7 +267,7 @@ func convertGoDef(gs *gog.Def, repoURI string) (*graph.Def, error) {
 	if err != nil {
 		return nil, err
 	}
-	path := string(pathOrDot(strings.Join(gs.Path, "/")))
+	path := string(pathOrDot(strings.Join(gs.Path, string(filepath.Separator))))
```

Comment:
`string(pathOrDot(strings.Join(gs.Path, string(filepath.Separator))))`

Can be made more simply:

`pathOrDot(filepath.Join(gs.Path...))`

There are 2 more instances like this below.

You could probably do this step via gofmt rewrite rules:

```bash
gofmt -w -r 'string(pathOrDot(strings.Join(x, string(filepath.Separator))))' -> 'pathOrDot(filepath.Join(x...))' ./
```